bridge/config/config.go

```diff
@@ -172,6 +173,7 @@ type ConfigValues struct {
 	General            Protocol
 	Gateway            []Gateway
 	SameChannelGateway []SameChannelGateway
+	ConfigFile         string
```

Comment:
This is not needed, matterbridge needs to run with `matterbridge -conf /tmp/matterbridge.toml` for example.
The configfile gets set on commandline.